tests/thread_local_large/thread_local_large.edl

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+enclave {
+#ifdef OE_SGX
```

Comment:
To allow the enclave to work with code coverage, we'd need to do this currently:

```suggestion
#ifdef OE_SGX
        from "openenclave/edl/fcntl.edl" import *;
```